ChatApp/Data/SignUp/SignUpState.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  SignUpResult.swift
+//  ChatApp
+//
+//  Created by Kalvin Dela Raga on 1/27/21.
+//
+
+import Foundation
+
+enum SignUpState {
```

Comment:
I think u can reuse this across all api calls, so make its name in geenral maybe just State or RequestState?